testing/mapserver/APKBUILD

```diff
@@ -1,18 +1,17 @@
 # Contributor: Bjoern Schilberg <bjoern@intevation.de>
 # Maintainer: Bjoern Schilberg <bjoern@intevation.de>
 pkgname=mapserver
-pkgver=7.2.1
+pkgver=7.4.0
 pkgrel=0
 pkgdesc="MapServer is CGI-based platform for publishing spatial data and interactive mapping applications to the web."
 url="http://mapserver.org/"
 arch="x86_64"
 license="MIT"
-depends="geos gdal proj4 cairo curl fcgi freetype libjpeg libpng libxml2 libpq giflib protobuf-c"
-depends_dev="cmake postgresql-dev geos-dev gdal-dev jpeg-dev giflib-dev libxml2-dev proj4-dev fcgi-dev curl-dev freetype-dev cairo-dev protobuf-c-dev"
+depends="geos gdal proj cairo curl fcgi freetype libjpeg libpng libxml2 libpq giflib protobuf-c"
```

Comment:
shouldn't depends be removed since depends_dev will find it automatically ?

Reply:
Yes, depends are automatically detected. I removed them.